components/badge/index.html

```diff
@@ -23,8 +23,8 @@
         <span class="badge-absolute badge-btn badge-bold badge-round badge-blue">23</span>
       </div>
       <div  class="badge-relative">
-        <button>Hide</button>
-        <span class="badge-absolute badge-btn-right badge-bold badge-round badge-blue">5</span>
+        <button onClick="myFunction()"id="btn-hide">Hide</button>
```

Comment:
You can use DOM to add functionality to this button, using DOM to add a click listener to this function is a better practice.

Reply:
`const btnInput=document.querySelector("#btn-input");
const hideBadge=()=>{
    const badge=document.querySelector("#badge-hide");
    if(badge.style.visibility === "visible"){
        badge.style.visibility="hidden";
    }else{
        badge.style.visibility="visible";
    }
}
btnInput.addEventListener("click",hideBadge);`

Reply:
okay i just updated have a look it